core/finality-grandpa/src/communication/mod.rs

```diff
@@ -64,7 +64,9 @@ mod cost {
 	pub(super) const BAD_SIGNATURE: i32 = -100;
 	pub(super) const MALFORMED_COMMIT: i32 = -1000;
 	pub(super) const FUTURE_MESSAGE: i32 = -500;
-	pub(super) const UNKNOWN_VOTER: i32 = -150;
+    pub(super) const UNKNOWN_VOTER: i32 = -150;
```

Comment:
```suggestion
	pub(super) const UNKNOWN_VOTER: i32 = -150;
```

Please remember to use tabs. This was brought up multiple times in review for this PR.